src/next-reminders/lib/db.ts

```diff
@@ -2,8 +2,11 @@ import { Pool, QueryConfig, QueryResult } from 'pg';
 import { runWithLogging } from '../../lib/db';
 import { getCurrentReminderPeriod } from './utils';
 
-export function getNextReminders(pool: Pool): Promise<QueryResult> {
-	const currentReminderPeriod = getCurrentReminderPeriod();
+export function getNextReminders(
+	pool: Pool,
+	now: Date = new Date(),
```

Comment:
Rather than mocking out `new Date()` it felt easiest to just make it an optional argument to the function

Reply:
👍 I much prefer passing in dependencies rather than mocks